Gems/SceneProcessing/Code/Source/SceneBuilder/SceneBuilderWorker.cpp

```diff
@@ -463,7 +463,7 @@ namespace SceneBuilder
         {
             AZStd::string productName;
             AzFramework::StringFunc::Path::GetFullFileName(scene->GetSourceFilename().c_str(), productName);
-            AzFramework::StringFunc::Path::ReplaceExtension(productName, "dbgsg");
+            productName += ".dbgsg";
```

Comment:
Does this require version bump in the asset builder?

Reply:
Technically _maybe_ it should, but since this isn't a true product asset and is just a set of debug files I'm not super worried about older versions lying around. It can eventually fix itself up as files get reprocessed. In the case where it was causing problems, you would've gotten a failure or a hung job, so those files would reprocess again anyways.

I'd hate to cause a full reprocess of every FBX file for everyone just because this extension changed, which *only* affects things if you have "Debug Output" turned on.